Automattic/vip-go-mu-plugins
# Fix phpcs issues for vip-init plugin

## Description

This PR fixes the issues found by `phpcs` in the `000-vip-init.php` plugin.

I will annotate all found issues except for the whitespace-related ones.

## Changelog Description

### Fix coding style and formatting for VIP Init

Fix phpcs errors and warnings for VIP Init

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Run `composer install`
2. Run `vendor/bin/phpcs -s 000-vip-init.php lib/ config/` and observe the reported issues
3. Check out the PR
4. Re-run `vendor/bin/phpcs -s 000-vip-init.php lib/ config/` and make sure that all previously reported issues are gone.
